steps/browserstack-app-live-upload/1.0.0/step.yml

```diff
@@ -0,0 +1,58 @@
+title: Browserstack App Live Upload
+summary: |
+  Uploads your APK or IPA to Browserstack for interactive app testing.
+description: "Uploads your APK or IPA to Browserstack for interactive app testing.
+  \nYou can find your username and access key at https://www.browserstack.com/app-live/rest-api\n"
+website: https://github.com/andrewmarmion/bitrise-step-browserstack-app-live-upload
+source_code_url: https://github.com/andrewmarmion/bitrise-step-browserstack-app-live-upload
+support_url: https://github.com/andrewmarmion/bitrise-step-browserstack-app-live-upload/issues
+published_at: 2019-10-09T21:13:42.734529+01:00
+source:
+  git: https://github.com/andrewmarmion/bitrise-step-browserstack-app-live-upload.git
+  commit: fbbb2908a9d69b2ac5215468736b2b49ac7a3e25
+host_os_tags:
+- osx-10.10
+- ubuntu-16.04
+- ios
```

Comment:
Starting from iOS these are project_type_tags, not host_os_tags, please modify step.yml accordingly.

Reply:
I have added the missing project_type_tages